configure.ac

```diff
@@ -335,7 +335,7 @@ AC_SUBST(CFG_CXXFLAGS_PROFILE)
 # c++14 is the oldest that Verilator supports
 # gnu is required for Cygwin to compile verilated.h successfully
 #_MY_CXX_CHECK_SET(CFG_CXXFLAGS_STD_NEWEST,-std=gnu++20)
-#_MY_CXX_CHECK_SET(CFG_CXXFLAGS_STD_NEWEST,-std=c++20)
+_MY_CXX_CHECK_SET(CFG_CXXFLAGS_STD_NEWEST,-std=c++20)
```

Comment:
We only require C++20, there's a later test for what's needed for coroutines - so should be added there presumably,

Reply:
Is there a reason c++20 std is not enabled by default? If there is, will that go under `AC_MSG_CHECKING([whether coroutines are supported by $CXX])`?

If not this shouldn't cause any problems with older stdlibs.

Reply:
We still want older compilers to work, and because choosing a std that does not match glibc often makes compiling break.   Ideally you wouldn't need that flag to use coroutines.

Reply:
Since they moved it from experimental/coroutine to coroutine without enabling c++20 std does not include coroutines. If we switch to c++17 we need "experimental/coroutine" but updated versions of the compiler does not have that.

So I don't see a simple way to make it work with both new and old compilers without enabling c++20. If there is any check I can make before enabling it (if we are going to enable it) let me know, I am not familiar with autoconf.

Reply:
Try instead adding this to the later section

_MY_CXX_CHECK_SET(CFG_CXXFLAGS_COROUTINES,-std=c++20)

Reply:
Added that to coroutine checks, enables c++20 with clang 17.0.6. Tested with multiple modules, seems like it's working.

Reply:
As above we cannot change NEWEST, please try the other flag suggested.  Also would really like a config fest instead of looking for a magic clang version which is likely fragile.

Reply:
> As above we cannot change NEWEST, please try the other flag suggested. Also would really like a config fest instead of looking for a magic clang version which is likely fragile.

We tried `_MY_CXX_CHECK_SET(CFG_CXXFLAGS_COROUTINES,-std=c++20)`, caused some gcc tests to fail. What should we test to enable c++20?